peer/brontide.go

```diff
@@ -1164,9 +1164,11 @@ func waitUntilLinkActive(p *Brontide,
 
 	// The link may already be active by this point, and we may have missed the
 	// ActiveLinkEvent. Check if the link exists.
-	link, _ := p.cfg.Switch.GetLink(cid)
-	if link != nil {
-		return link
+	links, _ := p.cfg.Switch.GetLinksByInterface(p.cfg.PubKeyBytes)
```

Comment:
Isn't `GetLink` already returning `ChannelUpdateHandler` now? Why do we switch to `GetLinksByInterface`?

Reply:
I wanted to use `GetLinksByInterface` in tests rather than `GetLink` to avoid repetition.  In the future we could even extend the Switch to get a list of links based on channel ids.